src/CommandLineUtilities/Convert.cxx

```diff
@@ -0,0 +1,44 @@
+///
+/// \file Convert.cxx
+/// \author Adam Wegrzynek <adam.wegrzynek@cern.ch>
+///
+
+#include "Configuration/ConfigurationFactory.h"
+#include <fstream>
+#include <boost/program_options.hpp>
+
+using namespace std::literals::string_literals;
+
+int main(int argc, char *argv[]) {
+  std::string sourceUri, destinationUri;
+  const std::string TEMP_FILE = "/tmp/alice_o2_configuration_convert_file.json";
```

Comment:
Why is a temporary file made necessary?

Reply:
Ah true, it's a work around of file reading in constructor, I'll get rid of it